src/olympia/reviewers/templates/reviewers/policy_view.html

```diff
@@ -1,7 +1,7 @@
 {% extends "reviewers/base.html" %}
 
 {% block title %}
-  {{ reviewer_page_title(page_title|format_html(addon=addon.name)) }}
+  {{ _('{page_title} – {addon} – Add-ons for Firefox')|format_html(addon=addon.name) }}
```

Comment:
this is causing a test error now in the reviewers/tests/test_views.py tests - `format_html` needs to be passed both  `page_title` and `addon`, just like `'{page_title} – {addon} – Add-ons for Firefox'.format(addon=addon.name)` would fail in python with the same error.